Added SQLinForm Plugin

guidomarcel opened the pull request:
Hi,
I added SQLinForm x64 plugin
Regards
Guido

guidomarcel committed:
Added SQLinForm Plugin

chcg commented on plugins/plugins64.xml:
```diff
@@ -341,6 +341,30 @@
+                <stability>Good</stability>
+                <install>
+                        <x64>
+                               <download>http://www.sqlinform.com/npp/SQLinFormNpp64.dll</download>
```
PluginManager currently requires that the dowload is a zip file.

guidomarcel replied:
OK, changed to zip file

chcg commented on plugins/plugins64.xml:
```diff
@@ -341,6 +341,30 @@
+                <install>
+                        <x64>
+                               <download>http://www.sqlinform.com/npp/SQLinFormNpp64.dll</download>
+                               <copy from="SQLinFormNpp64.dll" to="$PLUGINDIR$\" validate="true"/>
```
Valitate= true requires a md5 sum of the dll in https://github.com/bruderstein/npp-plugins-x64/blob/master/plugins/validate.json

guidomarcel replied:
OK, changed to validate="false"

guidomarcel committed:
download file now zip file

Hi,
download file is now a zip file and since I do not have a md5 I put validate="false"

chcg commented on plugins/plugins64.xml:
```diff
@@ -341,6 +341,30 @@
+                <stability>Good</stability>
+                <install>
+                        <x64>
+                               <download>http://www.sqlinform.com/npp/SQLinFormNpp64.zip</download>
```
@guidomarcel  Could you also provide the 32bit version as zip, so I could add it also to the list for 32bit plugins at https://npppm.bruderste.in/plugins

guidomarcel replied:
@Bruderstein,

OK. Zip file is available at http://www.sqlinform.com/npp/SQLinFormNpp.zip

 

All other info are identical to 64 bit version.

 

I will have a look to the version number and update it for the next release.

 

Also I will take care about md5.

 

Anything else I need to do for the 32 bit version?

 

Thanks and Regards

Guido

 

 

Von: chcg [mailto:notifications@github.com] 
Gesendet: Montag, 26. Juni 2017 17:54
An: bruderstein/npp-plugins-x64 <npp-plugins-x64@noreply.github.com>
Cc: guidomarcel <guido.thelen@gmail.com>; Mention <mention@noreply.github.com>
Betreff: Re: [bruderstein/npp-plugins-x64] Added SQLinForm Plugin (#21)

 

In plugins/plugins64.xml <https://github.com/bruderstein/npp-plugins-x64/pull/21#discussion_r124047743> :

> @@ -341,6 +341,30 @@
>                      </x64>
>              </install>
>       </plugin>
> +     <plugin name="SQLinForm SQL Formatter">
> +                <x64Version>5.6.63</x64Version>
> +                <aliases>
> +                     <alias name="SQLinForm"/>
> +                </aliases>
> +                <description>SQLinForm formats plain SQL, SQL embedded in Program Code, SQL snippets and SQL Statements with syntax errors.\n SQLinForm's Plugin for Notepad++ has a powerful code folding feature and formats your SQL as you Type.
> +                             It supports all major DB like Oracle, DB2, SQL Server, Teradata, Netezza, MySQL, Sybase, MS Access, etc. SQLinForm incorporates its own multi-DB parser engine. No additional software is required.</description>
> +                <author>Guido Thelen</author>
> +                <homepage>http://www.sqlinform.com <http://www.sqlinform.com%3c/homepage> </homepage>
> +                <latestUpdate>Added 64bit support</latestUpdate>
> +                <stability>Good</stability>
> +                <install>
> +                        <x64>
> +                               <download>http://www.sqlinform.com/npp/SQLinFormNpp64.zip <http://www.sqlinform.com/npp/SQLinFormNpp64.zip%3c/download> </download>

@guidomarcel <https://github.com/guidomarcel>  Could you also provide the 32bit version as zip, so I could add it also to the list for 32bit plugins at https://npppm.bruderste.in/plugins

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub <https://github.com/bruderstein/npp-plugins-x64/pull/21/files/bde03e8aca867ba0c7bcafc6fe7bb4311eb2fbad#r124047743> , or mute the thread <https://github.com/notifications/unsubscribe-auth/AcSRWqRgcUe3xcGk3UYGsW4Vw-NA91Olks5sH9P7gaJpZM4OEF0X> .  <https://github.com/notifications/beacon/AcSRWgO0sBxDYDDcyNTVi340iHPtHlASks5sH9P7gaJpZM4OEF0X.gif>

guidomarcel replied:
@bruderstein,

sonehow I am not able to create an account for  <https://npppm.bruderste.in/plugins> https://npppm.bruderste.in/plugins

 

Any idea?

 

Regards

Guido

 

Von: chcg [mailto:notifications@github.com] 
Gesendet: Montag, 26. Juni 2017 17:54
An: bruderstein/npp-plugins-x64 <npp-plugins-x64@noreply.github.com>
Cc: guidomarcel <guido.thelen@gmail.com>; Mention <mention@noreply.github.com>
Betreff: Re: [bruderstein/npp-plugins-x64] Added SQLinForm Plugin (#21)

 

In plugins/plugins64.xml <https://github.com/bruderstein/npp-plugins-x64/pull/21#discussion_r124047743> :

> @@ -341,6 +341,30 @@
>                      </x64>
>              </install>
>       </plugin>
> +     <plugin name="SQLinForm SQL Formatter">
> +                <x64Version>5.6.63</x64Version>
> +                <aliases>
> +                     <alias name="SQLinForm"/>
> +                </aliases>
> +                <description>SQLinForm formats plain SQL, SQL embedded in Program Code, SQL snippets and SQL Statements with syntax errors.\n SQLinForm's Plugin for Notepad++ has a powerful code folding feature and formats your SQL as you Type.
> +                             It supports all major DB like Oracle, DB2, SQL Server, Teradata, Netezza, MySQL, Sybase, MS Access, etc. SQLinForm incorporates its own multi-DB parser engine. No additional software is required.</description>
> +                <author>Guido Thelen</author>
> +                <homepage>http://www.sqlinform.com <http://www.sqlinform.com%3c/homepage> </homepage>
> +                <latestUpdate>Added 64bit support</latestUpdate>
> +                <stability>Good</stability>
> +                <install>
> +                        <x64>
> +                               <download>http://www.sqlinform.com/npp/SQLinFormNpp64.zip <http://www.sqlinform.com/npp/SQLinFormNpp64.zip%3c/download> </download>

@guidomarcel <https://github.com/guidomarcel>  Could you also provide the 32bit version as zip, so I could add it also to the list for 32bit plugins at https://npppm.bruderste.in/plugins

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub <https://github.com/bruderstein/npp-plugins-x64/pull/21/files/bde03e8aca867ba0c7bcafc6fe7bb4311eb2fbad#r124047743> , or mute the thread <https://github.com/notifications/unsubscribe-auth/AcSRWqRgcUe3xcGk3UYGsW4Vw-NA91Olks5sH9P7gaJpZM4OEF0X> .  <https://github.com/notifications/beacon/AcSRWgO0sBxDYDDcyNTVi340iHPtHlASks5sH9P7gaJpZM4OEF0X.gif>

guidomarcel replied:
@Bruderstein,

OK. Zip file is available at http://www.sqlinform.com/npp/SQLinFormNpp.zip

 

Anything else I need to do for the 32 bit version?

 

By the way. I cannot register at  <https://npppm.bruderste.in/plugins> https://npppm.bruderste.in/plugins . NO mail is sent to me.

 

Thanks and Regards

Guido

 

 

Von: chcg [mailto:notifications@github.com] 
Gesendet: Montag, 26. Juni 2017 17:54
An: bruderstein/npp-plugins-x64 <npp-plugins-x64@noreply.github.com>
Cc: guidomarcel <guido.thelen@gmail.com>; Mention <mention@noreply.github.com>
Betreff: Re: [bruderstein/npp-plugins-x64] Added SQLinForm Plugin (#21)

 

In plugins/plugins64.xml <https://github.com/bruderstein/npp-plugins-x64/pull/21#discussion_r124047743> :

> @@ -341,6 +341,30 @@
>                      </x64>
>              </install>
>       </plugin>
> +     <plugin name="SQLinForm SQL Formatter">
> +                <x64Version>5.6.63</x64Version>
> +                <aliases>
> +                     <alias name="SQLinForm"/>
> +                </aliases>
> +                <description>SQLinForm formats plain SQL, SQL embedded in Program Code, SQL snippets and SQL Statements with syntax errors.\n SQLinForm's Plugin for Notepad++ has a powerful code folding feature and formats your SQL as you Type.
> +                             It supports all major DB like Oracle, DB2, SQL Server, Teradata, Netezza, MySQL, Sybase, MS Access, etc. SQLinForm incorporates its own multi-DB parser engine. No additional software is required.</description>
> +                <author>Guido Thelen</author>
> +                <homepage>http://www.sqlinform.com <http://www.sqlinform.com%3c/homepage> </homepage>
> +                <latestUpdate>Added 64bit support</latestUpdate>
> +                <stability>Good</stability>
> +                <install>
> +                        <x64>
> +                               <download>http://www.sqlinform.com/npp/SQLinFormNpp64.zip <http://www.sqlinform.com/npp/SQLinFormNpp64.zip%3c/download> </download>

@guidomarcel <https://github.com/guidomarcel>  Could you also provide the 32bit version as zip, so I could add it also to the list for 32bit plugins at https://npppm.bruderste.in/plugins

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub <https://github.com/bruderstein/npp-plugins-x64/pull/21/files/bde03e8aca867ba0c7bcafc6fe7bb4311eb2fbad#r124047743> , or mute the thread <https://github.com/notifications/unsubscribe-auth/AcSRWqRgcUe3xcGk3UYGsW4Vw-NA91Olks5sH9P7gaJpZM4OEF0X> .  <https://github.com/notifications/beacon/AcSRWgO0sBxDYDDcyNTVi340iHPtHlASks5sH9P7gaJpZM4OEF0X.gif>
